scripts/fork_clone.py

```diff
@@ -0,0 +1,67 @@
+# Fork and Clone by Python
+
+import argparse
+import requests
+import json
+import os
+
+# Parse command-line arguments
+parser = argparse.ArgumentParser(description='Fork a repo')
+parser.add_argument('owner', type=str, help='Repository owner')
+parser.add_argument('repo', type=str, help='Repository name')
+parser.add_argument('token', type=str, help='GitHub API token')
+parser.add_argument('name',type=str,help='Enter name for forked repo')
+args = parser.parse_args()
+
+# Get the command-line arguments
+owner = args.owner
+repo = args.repo
+token = args.token
```

Comment:
we already have a github_secrets.py file in the project. import the token from there.

Reply:
I used the github_secrets.py to import the token
![image](https://github.com/sahil-sagwekar2652/GitHub-Automation-scripts/assets/85406626/cc037b8f-cbb6-46a9-a335-6f3b469e85a5)